src/pages/candidate-details/candidate-details.html

```diff
@@ -88,8 +87,7 @@ <h4>Candidate details</h4>
           </ion-col>
           <ion-col col-8>
             <div class="mes-data">
-              <div>Entitlement check is required.</div>
-              <div>Call deployment - 0191 201 8118</div>
+              <div>Entitlement check is required. Call deployment</div>
             </div>
           </ion-col>
         </ion-row>
```

Comment:
Can't spot changes 4 or 5 here on the image in Jira - https://jira.i-env.net/secure/thumbnail/56501/_thumb_56501.png

Reply:
I think this story only covers items 1-3 on that picture: https://jira.i-env.net/browse/MES-1738

Reply:
That's what happens when I skip to the pictures